### PR TITLE
For all application actions ID Check option is showing. ID check should be available for Certified and Corrected record action

### PR DESCRIPTION
**Pull request by favour234**

### CODE REVIEW

File: packages/client/src/views/RecordAudit/ActionDetailsModal.tsx [hunk 1]
```diff
@@ -424,7 +424,9 @@ export const ActionDetailsModalListTable = ({
       )}
 
       {/* Correction Requester Id Verified */}
-      {actionDetailsData.requester !== CorrectorRelationship.ANOTHER_AGENT &&
+      {(actionDetailsData.action == RegAction.RequestedCorrection ||
```
Comment (Zangetsu101) [inline packages/client/src/views/RecordAudit/ActionDetailsModal.tsx:427]:
```suggestion
      {(actionDetailsData.action === RegAction.RequestedCorrection ||
```

**Comment by Zangetsu101** [comment]
@favour234 I don't know why you have so many commits in this PR whereas only the last one seems to be related to the ticket

**Comment by favour234** [comment]
> @favour234 I don't know why you have so many commits in this PR whereas only the last one seems to be related to the ticket

i pulled updates from develop and new files shows as "modified" when i actually didn't modify ..i just pulled.

**Comment by Zangetsu101** [comment]
I'm guessing your local develop branch and the remote one aren't in sync with each other. I would suggest you make sure they are

**Pull request closed without merging**